package.json

```diff
@@ -32,7 +32,9 @@
     "@mdx-js/mdx/**/trim": "^0.0.3",
     "fs-extra": "^10.0.0",
     "next": "^11.0.1",
-    "ws": "^7.4.6"
+    "ws": "^7.4.6",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.5.6"
```

Comment:
Vue test utils doesn't work with Jest 27, so I had to lock this at 26. The React tests seems to still work with it, so this should be fine.